unlock-protocol-com/scripts/deploy-fleek.sh

```diff
@@ -6,13 +6,10 @@ set -e
 # This script deploys a static build to fleek.
 
 APP_PATH=$1
-DEPLOY_ENV=$2 #ignored in this script since unlock-protocol.com is always deployed to production
+DEPLOY_ENV=$2 
```

Comment:
do we set the env vars for staging/prod correctly?

Reply:
We don't actually use fleek anymore... so we can probably clean up that file.